templates/events/_pagination.html.twig

```diff
@@ -0,0 +1,34 @@
+<div class="l__wrapper--narrow text--body b__nudge--bottom-larger listing__paginator">
```

Comment:
Couldn't you add the route name as a variable of the inclusion here and not duplicate this template?